crates/sui-core/src/transaction_manager.rs

```diff
@@ -151,6 +185,9 @@ impl TransactionManager {
                 continue;
             }
             // skip already executed txes
+            // Tx lock is not held, which makes it possible to have multiple executions of
```

Comment:
can you clarify that this allows multiple concurrent enqueues, not multiple executions? the lock is still acquired during execution.

Reply:
Makes sense, updated.